examples/image-to-text/run_pipeline.py

```diff
@@ -101,6 +102,16 @@ def main():
         action="store_true",
         help="Whether to enable Habana Flash Attention in recompute mode on first token generation. This gives an opportunity of splitting graph internally which helps reduce memory consumption.",
     )
+    parser.add_argument(
+        "--use_kv_cache",
+        action="store_true",
+        help="Whether to use the key/value cache for decoding. It should speed up generation."
+    )
+    parser.add_argument(
+        "--profiling",
+        action="store_true",
+        help="enable profiling"
+    )
```

Comment:
```suggestion
    parser.add_argument(
        "--profiling_warmup_steps",
        default=0,
        type=int,
        help="Number of steps to ignore for profiling.",
    )
    parser.add_argument(
        "--profiling_steps",
        default=0,
        type=int,
        help="Number of steps to capture for profiling.",
    )
```

probably you can augment with other options for profiling!

Reply:
close this thread and using above one for profiling